From: Viswanath Srikanth Bathina <bathina@qti.qualcomm.com>
Date: Wed, 30 Sep 2026 15:41:40 +0530
Subject: [PATCH] common: add lib64 library search path

Add `/usr/lib64` to the default library search paths so camera-service can
locate its runtime libraries on systems that use the lib64 layout.

Upstream-Status: Pending
Signed-off-by: Viswanath Srikanth Bathina <bathina@qti.qualcomm.com>
---
 common/utils/qmmf_common_utils_defs.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/common/utils/qmmf_common_utils_defs.h b/common/utils/qmmf_common_utils_defs.h
index 5cef3a2..6584d7c 100644
--- a/common/utils/qmmf_common_utils_defs.h
+++ b/common/utils/qmmf_common_utils_defs.h
@@ -69,6 +69,7 @@ inline const char* kCameraMetaDataLibVersion = "1";
 
 #define LIB_SEARCH_PATH_DEFAULT   "/usr/lib"
 #define LIB_SEARCH_PATH_MULTIARCH "/usr/lib/aarch64-linux-gnu"
+#define LIB_SEARCH_PATH_LIB64      "/usr/lib64"
 
 enum class SocId {
   kInvalid = 0,
@@ -385,6 +386,7 @@ class Target {
     static const std::vector<std::string> search_paths = {
         LIB_SEARCH_PATH_DEFAULT,
         LIB_SEARCH_PATH_MULTIARCH,
+        LIB_SEARCH_PATH_LIB64,
     };
 
     for (const auto &path : search_paths) {
